### PR TITLE
feat: add MEME to eth lists

### PR DESCRIPTION
**Pull request by ChefBingbong**

### CODE REVIEW

File: src/tokens/pancakeswap-eth-mm.json [hunk 1]
```diff
@@ -134,5 +134,13 @@
     "chainId": 1,
     "decimals": 18,
     "logoURI": "https://tokens.pancakeswap.finance/images/eth/0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85.png"
+  },
+  {
```
Comment (chefjackson) [inline src/tokens/pancakeswap-eth-mm.json:138]: we should add to eth-default instead of eth-mm

Comment (ChefBingbong) [reply src/tokens/pancakeswap-eth-mm.json:138]: i asked momota on this he said to both, but can remove if u think its better

Comment (ChefMomota) [reply src/tokens/pancakeswap-eth-mm.json:138]: `src/tokens` also need add `pancakeswap-eth-default.json`

Comment (chefjackson) [reply src/tokens/pancakeswap-eth-mm.json:138]: I don't think mm support all the tokens. We better check if we have the token whitelisted on be before we add to the tokenlist

Comment (chefjackson) [reply src/tokens/pancakeswap-eth-mm.json:138]: this is the mm list not the mini program list

File: lists/pancakeswap-eth-mm.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "PancakeSwap Ethereum MM",
-  "timestamp": "2023-05-18T10:36:55.498Z",
+  "timestamp": "2023-11-03T09:51:41.715Z",
   "version": {
     "major": 0,
     "minor": 0,
-    "patch": 5
+    "patch": 7
```
Comment (ChefMomota) [inline lists/pancakeswap-eth-mm.json:7]: Remove all mm change

File: src/tokens/pancakeswap-eth-mm.json [hunk 3]
```diff
@@ -135,4 +135,4 @@
     "decimals": 18,
     "logoURI": "https://tokens.pancakeswap.finance/images/eth/0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85.png"
   }
-]
+]
```
Comment (ChefMomota) [inline src/tokens/pancakeswap-eth-mm.json:138]: same

**Review by chefjackson — APPROVED** [review]

**Pull request merged**